tests/services/github.service.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { GithubService } from '../../src/app/core/services/github.service';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../constants/githubissue.constants';
+import {
+  GITHUB_LABEL_FUNCTIONALITY_BUG,
+  GITHUB_LABEL_MEDIUM_SEVERITY,
+  GITHUB_LABEL_TEAM_LABEL,
+  GITHUB_LABEL_TUTORIAL_LABEL
+} from '../constants/githublabel.constants';
+const nock = require('nock');
+
+describe('GithubService', () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+  it('sucessfully returns a GitHub Issue upon issue creation', () => {
```

Comment:
For both 'it' blocks, we are only testing `createIssue` method of `GithubService` class. We can create another describe block `createIssue` to group these specs together.

Reply:
You can refer to https://github.com/CATcher-org/CATcher/pull/977!